packages/apps-config/src/endpoints/productionRelayKusama.ts

```diff
@@ -157,6 +157,14 @@ export function createKusama (t: TFunction): EndpointOption {
         providers: {
           StakeTechnologies: 'wss://rpc.shiden.plasmnet.io'
         }
+      },
+      {
+        info: 'moonriver',
+        paraId: 2023,
```

Comment:
```suggestion
        isUnreachable: true,
        paraId: 2023,
```

Extra flag until RPC is up. (More-or-less when WASM is available, if reachable then)